Datadog.Trace.sln

```diff
@@ -1455,6 +1461,7 @@ Global
 		{51AF3B29-D107-4E22-8A85-DF02A8A898E5} = {8683D82A-2BBE-4199-9C36-C59F48804F90}
 		{DE15D292-DC01-4AEA-B473-C2F2908341A7} = {0884B566-D22E-498C-BAA9-26D50ABCAE3A}
 		{959E9599-8D99-43BC-8038-B91F76179C1C} = {498A300E-D036-49B7-A43D-821D1CAF11A5}
+		{95DEBF4E-6017-40AD-9840-C7D112591228} = {9E5F0022-0A50-40BF-AC6A-C3078585ECAB}
```

Comment:
```suggestion
```

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/4422/commits/01fcf69054a323668f0be7f36bed0fcfef6943ac